common/content/buffer.js

```diff
@@ -415,6 +416,18 @@ const Buffer = Module("buffer", {
     set textZoom(value) { Buffer.setZoom(value, false); },
 
     /**
+     * @property {number} The current browser's zoom level, as a
+     *     percentage with 100 as 'normal'.
+     */
+    get zoomLevel() {
```

Comment:
By all means, it's not a big issue at all :-) Just thought I'd ask if you knew anything about it and possibly how to "fix" it.